local/README.md

```diff
@@ -0,0 +1,27 @@
+To work with a local dependent module, check out that module into this
+directory, and put the local replace into the main go.mod.
+
+Example:
+
+```bash
+cd local
+git clone  git@github.com:Bolodya1997/sdk.git
```

Comment:
Do we need this kind of link here?

Reply:
Ok with any, since it is example.

Reply:
Yep, it is an example of some custom import replace usage.